securitycenter/snippets_v2/snippets_notification_v2_test.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for snippets."""
+
+import os
+import uuid
+
+import backoff
+from google.api_core.exceptions import InternalServerError, NotFound, ServiceUnavailable
+from google.cloud import securitycenter_v2 as securitycenter_v2
+import pytest
+
+import snippets_notification_configs_v2
+
+ORG_ID = os.environ["GCLOUD_ORGANIZATION"]
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
```

Comment:
Update to match Java env variables.

Reply:
It is hard coded in Java and I can't give the name SCC_PROJECT_ORG_ID as we need both organization I'd and project I'd and above mentioned variable name is generic name.

Reply:
Change the variables to:

```
ORG_ID = os.environ["SCC_PROJECT_ORG_ID"]
PROJECT_ID = os.environ["SCC_PROJECT_ID"] 
```

Reply:
addressed